.github/workflows/tests.yml

```diff
@@ -31,15 +33,13 @@ jobs:
             - 6379:6379
 
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
 
       - name: Set up Ruby ${{ matrix.ruby-version }}
-        uses: ruby/setup-ruby@v1.97.0
+        uses: ruby/setup-ruby@v1
         with:
           ruby-version: ${{ matrix.ruby-version }}
-
-      - name: Install dependencies
-        run: bundle install
+          bundler-cache: true
```

Comment:
Use the `ruby/setup-ruby` action to run `bundle install`. This action includes a caching capability, so this should speed up the CI pipeline a little.

Reply:
I have seen some projects (with large build matrices) have **slower** builds due to bandwidth throttling GitHub imposes downloading from the cache, which have been resolved by disabling the cache.

With that said, it's not consistent (and likely affects larger caches, like a typical `node_modules` directory) so thank you for adding.